src/main/resources/config/application.properties

```diff
@@ -1,4 +1,5 @@
 # Connect Configuration
 connect.issuerUri = https://connect.ch.tudelft.nl/
 connect.clientUri = http://localhost:8080
+spring.template.cache = false
```

Comment:
Heeft dit geen negatieve effecten op de performance in productie straks? Kan er op internet zo 1-2-3 niks over vinden...